<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

<head>

<meta http-equiv="content-type" content="text/html; charset=utf-8" />

<meta name="description" content="Benchmark Analysis of Algorithms for Determining and Quantifying Full-length mRNA Splice Forms from RNA-Seq Data" />

<meta name="keywords" content="RNA-Seq" />

<meta name="author" content="Katharina Hayer" />

<link rel="stylesheet" type="text/css" href="style.css" media="screen" />
<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
<link rel="icon" href="images/favicon.ico" type="image/x-icon">

<title>BP2</title>

</head>

<body>


<div id="wrapper">

<?php include('includes/header.php'); ?>
<?php include('includes/nav.php'); ?>


<div id="content">

<h2>Datasets</h2>

<h3>Systematic dataset Test 1 (T1)</h3>

<p>
Our systematic dataset Test 1 (T1) has 13,000 (non-overlapping) genes with numbers of splice forms per gene varying from one to five. Each dataset has 50 million paired-end reads using mouse genome build mm9 (Mouse Genome Sequencing Consortium, et al. 2002).  Data set T1 was designed to determine upper bounds on the accuracies of the methods by providing data as ideal as possible.  Three different types of alternate splicing were included: (i) exon skipping, (ii) alternate 3’/5’ site (start/end of transcription) and (iii) alternate splice sites at the ends of internal exons.  The vast majority of alternate splicing events can be explained by these three categories.  The basic forms for dataset T1 were taken from real Ref-Seq genes (Pruitt KD, et al.  2014) as aligned to the reference genome by the UCSC genome browser (Kent WJ, et al. 2002).  The first 1,000 genes were chosen to have one splice form and at least 5 exons each.  For each of the three categories of alternate splicing there are an additional 4,000 genes with 1,000 having two forms, 1,000 having three forms, 1,000 having four forms and 1,000 having five forms.  In this data set we did not mix differ-ent types of alternate splicing in the same gene.  In other words, for each gene all forms differ by splicing types of just one of the three categories.  In total there are 13,000 genes comprising 43,000 splice forms. The alternate splice forms were generated by randomly removing some exons (exon skipping), or by randomly leaving exons off one or both ends of the tran-script (truncation), or by randomly altering the start or end splice site of individual exons (mostly by multiples of three as is typically observed in real data).  The reads for Test 1 were generated with the following parame-ters: read length=100, fragment length min=200, fragment length max=500, fragment length median=300, basewise error=0%, substitution frequen-cy=0%, indel frequency=0%, intron frequency=0%.  All transcripts were expressed at the same high level with at approximately 40X coverage. We take fragmentation into account yielding data that differ from strictly uni-form by the randomness of the fragmentation process.  The results of this data set should effectively give bounds on the accuracy of the various methods, since no method could do better in reality than it does on perfect data.
</p>
<h4>Download: <br>
<div id="under-element">
<center>
<a href="http://bp2.s3.amazonaws.com/hayer1_T1.tar.bz2">hayer1_T1.tar.bz2</a>
  (<a href="http://bp2.s3.amazonaws.com/hayer1_T1.tar.bz2.md5sum">hayer1_T1.tar.bz2.md5sum</a>) 
</center>
<span id="hover-element">
<table>
<tr><th>File name <th> File size in mb</tr>
<tr><td>hayer1_T1.tar.bz2 <td> 18,106.76</tr>
<tr><td>simulated_reads_test1.cig <td>16,518.19</tr>
<tr><td>simulator_config_geneinfo_test1 <td>9.42</tr>
<tr><td>simulator_config_featurequantifications_test1 <td>35.74</tr>
<tr><td>simulated_reads_test1.forward.fa<td>  5,520.71</tr>
<tr><td>simulated_reads_test1.reverse.fa<td>  5,198.00</tr>
<tr><td>simulated_reads_junctions-crossed_test1.txt<td> 1,234.61</tr>
<tr><td>fraglenhisto_test1.txt<td>  0.00</tr>
<tr><td>simulated_reads2genes_test1.txt<td> 1,188.07</tr>
<tr><td>simulated_reads_test1.log<td> 0.00</tr>
<tr><td>true_alignment.bam <td> 1,998.46</tr>
<tr><td>true_alignment.bam.bai <td> 2.67</tr>
<tr><td>tophat_without_gene_models.bam.bai<td>  3.49</tr>
<tr><td>tophat_with_gene_models.bam<td> 1,941.81</tr>
<tr><td>tophat_without_gene_models.bam <td> 1,893.04</tr>
<tr><td>tophat_with_gene_models.bam.bai<td> 2.88</tr>
<tr><td>star_without_gene_models.bam <td> 2,031.73</tr>
<tr><td>star_without_gene_models.bam.bai <td> 3.05</tr>
<tr><td>star_with_gene_models.bam <td>2,030.98</tr>
<tr><td>star_with_gene_models.bam.bai <td>3.05</tr>
</table>
</span>
</div>
</h4>

<h3>ENSEMBL Perfect (EP) and ENSEMBL Realistic (ER)</h3>

<p>
Two additional simulated data sets were generated to assess the effect of polymorphisms and sequencing artifacts.  Data set EP (ENSEMBL Perfect) was generated with the same parameters as T1, except the full set of 93,778 unmodified ENSEMBL transcript models was used.  Data set ER (ENSEMBL Realistic) was generated with the same parameters except for the following changes, chosen to mimic real data: basewise error=0,5%, substitution frequency=0.1%, indel frequency=0.05%, intron frequen-cy=30%.  These are fairly low polymorphism rates that would be expected if comparing human sequencing data to the human reference genome.  In reality the polymorphism rates in other organisms will be much higher.  We chose these parameters in order to attain reasonable lower bounds on algo-rithm accuracy in practice, where human/human comparisons are currently among the least polymorphic of all vertebrates.  Intron frequency of 30% may seem high at first, but it is typical.  However, in spite of comprising nearly a third of the reads, since introns are so much larger than exons the majority of intron signal results in very low coverage.  Two-thirds of all transcripts are expressed above zero, with levels of expressed transcripts given by an exponential distribution.  Short genes will be under-represented because of the fragment length distribution so we removed all genes under 200 bases.
</p>
<h4>Download: <br> 
<center>
<div id="under-element">
<a href="http://bp2.s3.amazonaws.com/hayer1_EP.tar.bz2">hayer1_EP.tar.bz2</a>
  (<a href="http://bp2.s3.amazonaws.com/hayer1_EP.tar.bz2.md5sum">hayer1_EP.tar.bz2.md5sum</a>),<br> 
<span id="hover-element">
<table>
<tr><th>File name <th> File size in mb</tr>
<tr><td>hayer1_EP.tar.bz2 <td>  17,870.41</tr>
<tr><td>simulated_reads_EP.cig <td> 16,487.73</tr>
<tr><td>simulated_reads_EP.forward.fa<td> 5,520.71</tr>
<tr><td>simulated_reads_EP.reverse.fa<td> 5,520.71</tr>
<tr><td>simulator_config_featurequantifications_ensembl-mm9_EP<td>  53.52</tr>
<tr><td>simulated_reads_EP.log<td>  0.00</tr>
<tr><td>true_alignment.bam<td>  1,966.32</tr>
<tr><td>simulator_config_geneinfo_ensembl-mm9 <td>14.96</tr>
<tr><td>fraglenhisto_EP.txt<td> 0.00</tr>
<tr><td>simulated_reads_junctions-crossed_EP.bed <td> 13.87</tr>
<tr><td>simulated_reads2genes_EP.txt <td> 1,126.62</tr>
<tr><td>true_alignment.bam.bai <td> 3.00</tr>
<tr><td>tophat_without_gene_models.bam<td>  2,047.93</tr>
<tr><td>tophat_without_gene_models.bam.bai<td>  4.15</tr>
<tr><td>tophat_with_gene_models.bam<td> 1,926.33</tr>
<tr><td>tophat_with_gene_models.bam.bai<td> 3.12</tr>
<tr><td>star_without_gene_models.bam <td> 1,992.92</tr>
<tr><td>star_without_gene_models.bam.bai <td> 3.21</tr>
<tr><td>star_with_gene_models.bam <td>1,992.92</tr>
<tr><td>star_with_gene_models.bam.bai <td>3.21</tr>
</table>
</span>
</div>
<div id="under-element">
   <a href="http://bp2.s3.amazonaws.com/hayer1_ER.tar.bz2">hayer1_ER.tar.bz2</a>
  (<a href="http://bp2.s3.amazonaws.com/hayer1_ER.tar.bz2.md5sum">hayer1_ER.tar.bz2.md5sum</a>) 
<span id="hover-element">
<table>
<tr><th>File name <th> File size in mb</tr>
<tr><td>hayer1_ER.tar.bz2<td> 23,674.97</tr>
<tr><td>simulated_reads_ER.cig<td>  16,148.54 </tr>
<tr><td>fraglenhisto_ER.txt<td> 0.00 </tr>
<tr><td>simulated_reads_ER.log<td>  0.00 </tr>
<tr><td>simulated_reads_indels_ER.txt<td> 1.81 </tr>
<tr><td>simulated_reads_substitutions_ER.txt<td>  4.18 </tr>
<tr><td>simulator_config_geneinfo_ensembl-mm9 <td>14.96 </tr>
<tr><td>simulated_reads2genes_ER.txt <td> 1,125.66 </tr>
<tr><td>simulated_reads_ER.forward.fa <td>5,520.71 </tr>
<tr><td>simulated_reads_ER.reverse.fa <td>5,520.71 </tr>
<tr><td>simulated_reads_junctions-crossed_ER.bed <td> 13.78 </tr>
<tr><td>simulator_config_featurequantifications_ensembl-mm9_ED <td> 53.51 </tr>
<tr><td>true_alignment.bam <td> 2,754.96 </tr>
<tr><td>true_alignment.bam.bai <td> 4.00 </tr>
<tr><td>tophat_without_gene_models.bam <td> 3,561.19 </tr>
<tr><td>tophat_without_gene_models.bam.bai <td> 5.78 </tr>
<tr><td>tophat_with_gene_models.bam <td>3,228.77 </tr>
<tr><td>tophat_with_gene_models.bam.bai<td> 5.45 </tr>
<tr><td>star_without_gene_models.bam <td> 3,138.41 </tr>
<tr><td>star_without_gene_models.bam.bai <td> 5.17 </tr>
<tr><td>star_with_gene_models.bam <td>3,138.41 </tr>
<tr><td>star_with_gene_models.bam.bai <td>5.17 </tr>
</table>
</span>
</div>
</center>
</h4>
<h3>In vitro transcription dataset (IVT)</h3>
<p>
Finally, we used data from in vitro transcription of 1062 human cDNAs (IVT), sequenced by both poly A seq and total RNA-seq <a href="http://www.ncbi.nlm.nih.gov/pubmed/24981968">(Lahens et al. 2014)</a>. Because these are cDNAs, we know the exact nucleotide sequence, including all the exon-exon junctions. Of these genes 50 have multiple splice forms. This data provides an evaluation of each algorithm on real data.
</p>
<h4>Download: 
<center>
<div id="under-element">
<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50445">GEO ID=GSE50445</a> 
<span id="hover-element">
In particular: <br>
<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219408">IVT polyA selected</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219398">IVT Ribo selected 1</a> and <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219399">IVT Ribo selected 2</a>
</span>
</div>
<div id="under-element"> 


<a href="http://bp2.s3.amazonaws.com/hayer1_IVT.tar.bz2">hayer1_IVT.tar.bz2</a> 
  (<a href="http://bp2.s3.amazonaws.com/hayer1_IVT.tar.bz2.md5sum">hayer1_IVT.tar.bz2.md5sum</a>)

<span id="hover-element">
<table>
<tr><th>File name <th> File size in mb</tr>
<tr><td>hayer1_IVT.tar.bz2 <td> 78,390.47</tr>
<tr><td>IVT_polyA_star_with_gene_models.bam <td>198.87</tr>
<tr><td>IVT_polyA_star_with_gene_models.bam.bai<td> 3,312.47</tr>
<tr><td>IVT_polyA_star_without_gene_models.bam <td> 1,095.67</tr>
<tr><td>IVT_polyA_star_without_gene_models.bam.bai <td> 0.00</tr>
<tr><td>IVT_polyA_tophat_with_gene_models.bam <td>1,095.67</tr>
<tr><td>IVT_polyA_tophat_with_gene_models.bam.bai<td> 3,792.72</tr>
<tr><td>IVT_polyA_tophat_without_gene_models.bam<td>  0.69</tr>
<tr><td>IVT_polyA_tophat_without_gene_models.bam.bai <td> 0.19</tr>
<tr><td>IVT_ribo_star_with_gene_models.bam <td> 309.98</tr>
<tr><td>IVT_ribo_star_with_gene_models.bam.bai <td> 1.48</tr>
<tr><td>IVT_ribo_star_without_gene_models.bam <td>337.19</tr>
<tr><td>IVT_ribo_star_without_gene_models.bam.bai <td>1.65</tr>
<tr><td>IVT_ribo_tophat_with_gene_models.bam <td> 324.23</tr>
<tr><td>IVT_ribo_tophat_with_gene_models.bam.bai <td> 339.59</tr>
<tr><td>IVT_ribo_tophat_without_gene_models.bam <td>1.51</tr>
<tr><td>IVT_ribo_tophat_without_gene_models.bam.bai<td> 339.52</tr>
<tr><td>simulated_reads_spikeins.cig <td> 1.51</tr>
<tr><td>simulated_reads_spikeins.forward.fa <td>10,154.08</tr>
<tr><td>simulated_reads_spikeins.log<td>  1.66</tr>
<tr><td>simulated_reads_spikeins.reverse.fa<td> 11,641.36</tr>
<tr><td>simulated_reads_spikeins.sam<td>  2.39</tr>
<tr><td>simulated_reads2genes_spikeins.txt<td>  7,053.00</tr>
<tr><td>simulator_config_featurequantifications_spikeins_OLD<td>  1.68</tr>
<tr><td>simulator_config_geneinfo_spikeins <td> 8,167.85</tr>
<tr><td>star_with_gene_models.bam<td> 2.19</tr>
<tr><td>star_with_gene_models.bam.bai<td> 10,241.58</tr>
<tr><td>star_without_gene_models.bam<td>  1.89</tr>
<tr><td>star_without_gene_models.bam.bai<td>  11,637.21</tr>
<tr><td>tophat_with_gene_models.bam<td> 2.38</tr>
<tr><td>tophat_without_gene_models.bam<td>  7,093.60</tr>
<tr><td>tophat_without_gene_models.bam.bai <td> 1.85</tr>
<tr><td>true_alignment.bam  8,164.35</tr>
<tr><td>true_alignment.bam.bai  2.19</tr>
</table>
</span>
</div>
</center>
</h4>

<!-- In particular: <br>
<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219408">IVT polyA selected</a>, <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219398">IVT Ribo selected 1</a> and <a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM1219399">IVT Ribo selected 2</a>
-->
</div> <!-- end #content -->


<?php include('includes/footer.php'); ?>

</div> <!-- End #wrapper -->

  </body>

</html>
